reo/tests/test_chp.py

```diff
@@ -61,3 +61,63 @@ def test_chp_sizing(self):
             print("Error message: {}".format(d['messages'].get('error')))
             raise
 
+    def test_cost_curve(self):
+        """
+        Validation to ensure that:
+         1) CHP capital cost curve handling is correct
+         2)
+
+        :return:
+        """
+
+        # Call API, get results in "d" dictionary
+        nested_data = json.load(open(self.test_post, 'rb'))
+        # CHP
+        nested_data["Scenario"]["Site"]["CHP"] = {}
+        nested_data["Scenario"]["Site"]["CHP"]["prime_mover"] = "recip_engine"
+        nested_data["Scenario"]["Site"]["CHP"]["size_class"] = 2
+        nested_data["Scenario"]["Site"]["CHP"]["min_kw"] = 800
+        nested_data["Scenario"]["Site"]["CHP"]["max_kw"] = 800
+        nested_data["Scenario"]["Site"]["CHP"]["installed_cost_us_dollars_per_kw"] = [2900, 2700, 2370]
+        nested_data["Scenario"]["Site"]["CHP"]["tech_size_for_cost_curve"] = [100, 600, 1140]
+        #nested_data["Scenario"]["Site"]["CHP"]["utility_rebate_us_dollars_per_kw"] = 50
+        #nested_data["Scenario"]["Site"]["CHP"]["utility_rebate_max_us_dollars"] = 1.0E10 #50 * 200
+        #nested_data["Scenario"]["Site"]["CHP"]["federal_rebate_us_dollars_per_kw"] = 50
+        nested_data["Scenario"]["Site"]["CHP"]["federal_itc_pct"] = 0.1
+        nested_data["Scenario"]["Site"]["CHP"]["macrs_option_years"] = 0
+        nested_data["Scenario"]["Site"]["CHP"]["macrs_bonus_pct"] = 0
+        nested_data["Scenario"]["Site"]["CHP"]["macrs_itc_reduction"] = 0.0
+
+        init_capex_chp_expected = 2020666.67
```

Comment:
@Bill-Becker could you please add a comment for how you arrived at this expected value? (I tried the slope * size but there must be a y-intercept?)

Reply:
Addressed in this [commit](https://github.com/NREL/REopt_Lite_API/pull/112/commits/466a2f9d85a91e0ec39b58707fae4cf6459bb5d5)